docs/content/drafts/PageLayout.mdx

```diff
@@ -252,7 +252,7 @@ See [storybook](https://primer.style/react/storybook?path=/story/layout-pagelayo
     noUnnecessaryDeps: true,
     adaptsToThemes: true,
     adaptsToScreenSizes: true,
-    fullTestCoverage: false,
+    fullTestCoverage: true,
```

Comment:
Solid work here!

Would be nice to get more coverage on the branches here with all the different options

<img width="997" alt="image" src="https://user-images.githubusercontent.com/1863771/154484390-794000bb-8b5d-410a-b0f6-590fdde5c114.png">

Reply:
Adds more tests to cover these branches 👍  There are a few uncovered branches in some internal components (dividers) but I'd have to write some hacky code to cover those cases. Let me know if you comfortable with this level of code coverage